cli/dcoscli/data/config-schema/core.json

```diff
@@ -9,7 +9,7 @@
     },
     "dcos_url": {
       "type": "string",
-      "pattern": "^(?:(?:https?)://)(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\\.?)+(?:[a-zA-Z]{2,6}\\.?|[a-zA-Z0-9-]{2,}\\.?)?|\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})(?::\\d+)?(?:/?|[/?]\\S+)$",
+      "pattern": "^(?:(?:https?)://)(([a-zA-Z0-9_-])+(:[a-zA-Z0-9_-]+)?@){0,1}(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\\.?)+(?:[a-zA-Z]{2,6}\\.?|[a-zA-Z0-9-]{2,}\\.?)?|\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})(?::\\d+)?(?:/?|[/?]\\S+)$",
```

Comment:
Having said that I would be very happy if we could get rid of this patter and leverage jsonschema `url` pattern or Python's url parser.